frequency-chain/frequency-xyz
# feat: phone and table rendering for main DevPortal page

# Description

Render responsive design for phone & tablet.

- Closes #33 

## Screenshot(s)
<img width="378" alt="image" src="https://github.com/user-attachments/assets/0535423c-2327-42fb-9d10-a9d4517a7125">

<img width="378" alt="image" src="https://github.com/user-attachments/assets/36297975-8722-47a1-9da6-3b66b253f196">

<img width="378" alt="image" src="https://github.com/user-attachments/assets/7c371658-3718-4272-a358-304585d4fbeb">



# How to Test?

1. Pull and run locally
2. Go to "Developer Portal" page
3. See that the design at different sizes responds appropriately

# Checklist:
